src/core/MOM_open_boundary.F90

```diff
@@ -1127,26 +1127,30 @@ subroutine initialize_obc_tides(OBC, US, param_file)
   type(astro_longitudes) :: nodal_longitudes  !< Solar and lunar longitudes for tidal forcing
   type(time_type) :: nodal_time               !< Model time to calculate nodal modulation for.
   integer :: c                                !< Index to tidal constituent.
+  logical :: tides                            !< True if astronomical tides are also used.
 
   call get_param(param_file, mdl, "OBC_TIDE_CONSTITUENTS", tide_constituent_str, &
       "Names of tidal constituents being added to the open boundaries.", &
       fail_if_missing=.true.)
 
-  call get_param(param_file, mdl, "OBC_TIDE_ADD_EQ_PHASE", OBC%add_eq_phase, &
+  call get_param(param_file, mdl, "TIDES", tides, &
+      "If true, apply tidal momentum forcing.", default=.false., do_not_log=.true.)
+
+  call get_param(param_file, mdl, "TIDE_USE_EQ_PHASE", OBC%add_eq_phase, &
       "If true, add the equilibrium phase argument to the specified tidal phases.", &
-      default=.false., fail_if_missing=.false.)
+      default=.false., do_not_log=tides)
```

Comment:
I believe that in this and the next 3 get_param calls, the logic of the`do_not_log` flag is reversed.  We want to log these parameters if the tides are enabled, but not to log them if the tides are not being used.  In this case, we would want to change the final argument here and on lines 1145, 1149 and 1153 to `do_not_log=.not.tides`.

Reply:
My impression is that the OBC tidal forcing can be turned on even if the astronomical tidal forcing is not turned on. @andrew-c-ross please correct me if I'm wrong. But if this is the case, then `tides` will be `.false.` but we still want to log these parameters. If the astronomical tidal forcing is turned on, then these parameters will be logged by `MOM_tidal_forcing` and we don't need to log them again here.

Reply:
Thank you for that response.  I think that now I understand why this optional do_not_log is needed and the correct logic with which they are being set.

   We never log the same parameter twice, but because these calls occur in a routine (`initialize_obc_tides()`) that is called (via `open_boundary_config()` and `MOM_initialize_fixed()`) before `tidal_forcing_init()` (which is called via `initialize_dyn_split_RK2()` or a similar routine), these parameters would be logged as OBC parameters and not tidal parameters, which does not make sense.